src/download.jl

```diff
@@ -198,10 +198,18 @@ function downloadpdb(pdbid::AbstractString;
             # Download the compressed PDB file to the temporary location
             @info "Downloading file from PDB: $pdbid"
             if ba_number == 0
-                Downloads.download(
-                    "http://files.rcsb.org/download/$pdbid.$(pdbextension[format]).gz",
-                    archivefilepath,
-                )
+                if format == BCIFFormat
+                    Downloads.download(
+                        "https://models.rcsb.org/$pdbid.bcif",
+                        pdbpath,
+                    )
+                    return pdbpath
```

Comment:
Might be better not to return from within the `try` block, but to use the return at the end of the function and add `BCIFFormat` checks as required.